addons/test_mail_full/tests/test_mass_mailing_server.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.test_mail_full.tests.common import TestMailFullCommon
+from odoo.addons.base.tests.common import MockSmtplibCase
+from odoo.tests.common import users, MockValidator
+from odoo.tools import mute_logger
+
+
```

Comment:
HumanLinter: odoo.tests import should be located before from odoo.tests.common import users, MockValidator